backward.hpp

```diff
@@ -1737,59 +1747,97 @@ class Printer {
 
 	template <typename ST>
 		FILE* print(ST& st, FILE* os = stderr) {
+			std::stringstream ss;
+			Colorize colorize(ss);
+			if (color) {
+				colorize.activate_if_tty(os);
+			}
+			print(st, ss, colorize);
+			fprintf(os, "%s", ss.str().c_str());
```

Comment:
let's avoid using a temporary buffer, if anything crashes, we won't even get any partial output. When this code is executed in a signal handler context, many things can fail. Yes right now, most things will fail already, but the less the better.

It might be possible to instantiate a `std::ostream` from a `FILE*`, but I have no idea how to do it.